diff -ru php-5.3.3/ltmain.sh php-5.3.3.entropy/ltmain.sh
--- php-5.3.3/ltmain.sh	2009-11-23 22:53:30.000000000 +0100
+++ php-5.3.3.entropy/ltmain.sh	2009-11-23 22:53:30.000000000 +0100
@@ -60,6 +60,7 @@
 BIN_SH=xpg4; export BIN_SH # for Tru64
 DUALCASE=1; export DUALCASE # for MKS sh
 
+echo="$ECHO"
 # Check that we have a working $echo.
 if test "X$1" = X--no-reexec; then
   # Discard the --no-reexec flag, and continue.
