diff -ru php-5.3.3/ext/tidy/tidy.c php-5.3.3.entropy/ext/tidy/tidy.c 
--- php-5.3.3/ext/tidy/tidy.c	2010-03-12 11:28:59.000000000 +0100
+++ php-5.3.3.entropy/ext/tidy/tidy.c	2010-03-12 11:28:59.000000000 +0100
@@ -22,6 +22,8 @@
 #include "config.h"
 #endif
 
+#include "tidy.h"
+
 #include "php.h"
 #include "php_tidy.h"
 
@@ -31,7 +33,6 @@
 #include "ext/standard/info.h"
 #include "safe_mode.h"
 
-#include "tidy.h"
 #include "buffio.h"
 
 /* compatibility with older versions of libtidy */
